includes/class-wc-gutenberg-emails-loader.php

```diff
@@ -54,10 +54,41 @@ public function init_post_type() {
 	}
 
 	/**
-	 * Create emails content
+	 * Create email template posts.
 	 */
-	public function create_emails() {
-		// @todo
+	public function create_templates() {
+		global $wpdb, $pagenow;
+
+		// Only run this on the WooCommerce Emails page.
```

Comment:
Actually, if the plan is to remove the PHP templates, shouldn't we run this when the plugin is enabled? Otherwise, if the user doesn't visit the WooCommerce Emails page there would be no email templates created, right?

Reply:
Oh good question.  Is the plan to fully phase out the PHP templates and only rely on these?

I was thinking these would be secondary to the PHP templates only if published.